syntax = "proto3";

package dml;

message NodeDef {
  string name = 1;
  
  string op = 2;

  repeated string input = 3;

  repeated string output = 4;

  string args = 5;
}